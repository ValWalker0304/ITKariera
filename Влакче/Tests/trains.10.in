Add 73105 Zimnitsa F 40
Add 39098 Plovdiv P 9
Add 27720 Sindel P 1
Add 55598 Burgas F 13
Add 24555 Shumen P 10
History
Travel
Next
Next
Add 79862 Montana F 16
History
Add 17676 Burgas F 7
Travel
History
Travel
History
History
History
History
Travel
History
Add 66541 GOryahovitsa P 7
Next
Next
Next
History
Add 97501 Vratsa F 40
History
History
History
Add 85609 Burgas F 14
Add 73858 Targovishte P 10
Next
History
Travel
Next
Add 27980 Sindel F 32
History
History
Travel
Next
Add 67317 Blagoevgrad P 6
Travel
Travel
Travel
Travel
Next
Add 93528 Varna F 15
Next
Travel
Travel
Travel
Add 85330 Sliven P 9
Add 64069 Ruse P 5
History
Next
Next
Travel
Add 14556 Blagoevgrad F 20
Travel
Add 55159 Vratsa F 40
History
History
Next
Add 91067 Karlovo F 32
Next
Travel
Next
History
History
Add 38828 Zimnitsa P 2
History
Next
Travel
History
End