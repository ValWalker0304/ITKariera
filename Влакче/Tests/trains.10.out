24555 Shumen P 10
27720 Sindel P 1
27720 Sindel P 1
24555 Shumen P 10
27720 Sindel P 1
27720 Sindel P 1
24555 Shumen P 10
39098 Plovdiv P 9
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
17676 Burgas F 7
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
79862 Montana F 16
79862 Montana F 16
79862 Montana F 16
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
73858 Targovishte P 10
66541 GOryahovitsa P 7
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
27980 Sindel F 32
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
66541 GOryahovitsa P 7
85609 Burgas F 14
97501 Vratsa F 40
79862 Montana F 16
93528 Varna F 15
93528 Varna F 15
79862 Montana F 16
55598 Burgas F 13
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
73105 Zimnitsa F 40
73105 Zimnitsa F 40
73105 Zimnitsa F 40
14556 Blagoevgrad F 20
14556 Blagoevgrad F 20
73105 Zimnitsa F 40
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
14556 Blagoevgrad F 20
73105 Zimnitsa F 40
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
55159 Vratsa F 40
91067 Karlovo F 32
91067 Karlovo F 32
55159 Vratsa F 40
91067 Karlovo F 32
14556 Blagoevgrad F 20
73105 Zimnitsa F 40
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
91067 Karlovo F 32
14556 Blagoevgrad F 20
73105 Zimnitsa F 40
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
91067 Karlovo F 32
14556 Blagoevgrad F 20
73105 Zimnitsa F 40
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
55159 Vratsa F 40
55159 Vratsa F 40
55159 Vratsa F 40
91067 Karlovo F 32
14556 Blagoevgrad F 20
73105 Zimnitsa F 40
55598 Burgas F 13
79862 Montana F 16
93528 Varna F 15
97501 Vratsa F 40
85609 Burgas F 14
66541 GOryahovitsa P 7
67317 Blagoevgrad P 6
27980 Sindel F 32
73858 Targovishte P 10
17676 Burgas F 7
39098 Plovdiv P 9
27720 Sindel P 1
24555 Shumen P 10
